lib/react_client.dart

```diff
@@ -20,7 +21,7 @@ import 'package:react/src/typedefs.dart';
 
 export 'package:react/react_client/react_interop.dart' show ReactElement, ReactJsComponentFactory;
 
-final EmptyObject emptyJsMap = new EmptyObject();
+final dynamic emptyJsMap = newObject();
```

Comment:
I was initially thinking that this should be okay due to how loosey-goosey the typing of JS interop objects are.

However, I'm not sure this will work with strong mode... Hmm. Maybe it'd be best to just deprecate this field.